glocaltokens/client.py

```diff
@@ -47,34 +52,35 @@ def __init__(
         LOGGER.debug("Initializing new Device instance.")
         self.device_name = device_name
         self.local_auth_token = None
-        self.ip = ip
+        self.ip_address = ip_address
         self.port = port
         self.google_device = google_device
         self.hardware = hardware
 
         LOGGER.debug(
-            'Set self device_name to "{}", local_auth_token to None, '
-            "google_device to {}, and hardware to {}".format(
-                device_name, google_device, hardware
-            )
+            "Set self device_name to '%s', local_auth_token to None, "
+            "google_device to %s, and hardware to %s",
+            device_name,
+            google_device,
+            hardware,
         )
 
         if google_device:
             LOGGER.debug("google_device is not None")
-            LOGGER.debug(f"Setting self ip to {google_device.ip}")
-            self.ip = google_device.ip
-            LOGGER.debug(f"Setting self port to {google_device.port}")
+            LOGGER.debug("Setting self ip to %s", google_device.ip_address)
+            self.ip_address = google_device.ip_address
+            LOGGER.debug("Setting self port to %s", google_device.port)
             self.port = google_device.port
         else:
             LOGGER.debug("google_device is None")
-            if (ip and not port) or (not ip and port):
+            if (ip_address and not port) or (not ip_address and port):
```

Comment:
Let's cover all cases.
```suggestion
            if not ip_address or not port:
```

Reply:
Agree that the logic here can be improved, but let's keep it for another PR, cause it will also involve changing tests